libexec/pyenv-init

```diff
@@ -165,7 +165,7 @@ function help_() {
       fi
       echo
       echo 'export PYENV_ROOT="$HOME/.pyenv"'
-      echo '[[ -d $PYENV_ROOT/bin ]] && export PATH="$PYENV_ROOT/bin:$PATH"'
+      echo '[[ -d "$PYENV_ROOT/bin" ]] && export PATH="$PYENV_ROOT/bin:$PATH"'
```

Comment:
Double brackets lift the requirement for quoting

Reply:
Sorry, I don't understand your comment

Reply:
Inside double brackets, there's no need to quote variables by default